src/components/Menubar.js

```diff
@@ -33,21 +33,21 @@ class Menubar extends Component {
             <Menu pointing secondary>
                 <Menu.Item
                 name='dashboard'
-                active={activeItem === 'dashboard'}
+                active={activeItem === 'dashboard' || activeItem === '/dashboard'}
```

Comment:
```suggestion
                active={activeItem === 'dashboard' || activeItem === '/dashboard' || activeItem === '/'}
```
On this url http://localhost:3000/, it must show `dashboard` as active tab.